sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapp/AADAccessTokenGroupRolesExtractionTest.java

```diff
@@ -105,6 +106,15 @@ public void testGroupsNameAndGroupsId() {
         assertThat(groupsName).hasSize(2);
     }
 
+    @Test
+    public void testEnableFullList(){
+        setup();
+        when(properties.getUserGroup().getEnableFullList()).thenReturn(true);
+        Set<String> groupId= userService.extractGroupRolesFromAccessToken(accessToken);
+        assertThat(groupId).hasSize(2);
```

Comment:
I think we should have two tests
one for getEnableFullList==true, and another for getEnableFullList==false